src/ensembl/src/content/app/entity-viewer/hooks/useEntityViewerAnalytics.ts

```diff
@@ -92,11 +101,70 @@ const useEntityViewerAnalytics = () => {
     });
   };
 
+  const trackExternalLinkClick = (category: string, label: string) => {
+    analyticsTracking.trackEvent({
+      category,
+      label,
+      action: 'external_link_click'
+    });
+  };
+
+  const trackExternalLinkClickInProteinsList = (label: string) => {
+    trackExternalLinkClick('gene_view_proteins_list', label);
+  };
+
+  const trackProteinInfoToggle = (params: {
+    transcriptQuality: string | null;
+    transcriptId: string;
+    action: 'open_accordion' | 'close_accordion';
+    transcriptPosition: number;
+  }) => {
+    const { transcriptId, transcriptQuality } = params;
+    const label = transcriptQuality
+      ? `${transcriptQuality} ${transcriptId}` // "MANE Plus Clinical ENST00000380152.8"
+      : transcriptId;
+    analyticsTracking.trackEvent({
+      category: 'gene_view_proteins_list',
+      label,
+      action: params.action,
+      value: params.transcriptPosition + 1
+    });
+  };
+
+  const trackDownload = (params: TrackDownloadPayload) => {
+    const selectedOptions = params.options
```

Comment:
we decided to use prefix before the selected options
e.g. "Gene: BRCA2, Transcript: ENST00000380152.8, gene_genomic_sequence, transcript_cdna, transcript_protein_sequence"